src/test/ui/lint/lint-non-camel-case-types.stderr

```diff
@@ -1,74 +1,56 @@
-error: type `ONE_TWO_THREE` should have a camel case name
+error: type `ONE_TWO_THREE` should have an upper camel case name
   --> $DIR/lint-non-camel-case-types.rs:4:8
    |
 LL | struct ONE_TWO_THREE;
-   |        ^^^^^^^^^^^^^ help: convert the identifier to camel case: `OneTwoThree`
+   |        ^^^^^^^^^^^^^ help: convert the identifier to upper camel case: `ONETWOTHREE`
```

Comment:
This is incorrect. Also, I don't know if referring to "upper camel case" is better than "PascalCase".

Reply:
"upper camel case" was discussed in https://github.com/rust-lang/rfcs/pull/2389.

Reply:
Fixed the incorrect suggestion.